pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -16,6 +18,7 @@ var (
 	bootstrapInPlaceEnabledServices = []string{
 		"install-to-disk.service",
 	}
+	installDiskValidationRegex = regexp.MustCompile(`^(/([A-Za-z0-9_-]+)|/)+$`)
```

Comment:
This will work for:
`/dev/sda`
and 
`/dev/disk/by-uuid/7b96e627-3cb8-4036-a26a-f52b7b80f854`

But won't work for `/dev/disk/by-path/pci-0000:88:00.0-nvme-1`
Consider:
```suggestion
	installDiskValidationRegex = regexp.MustCompile(`^(/([A-Za-z0-9:._-]+)|/)+$`)
```

Reply:
thks @eranco74, good catch!